packages/web-vite/src/pages/LabelsPage.tsx

```
@@ -135,156 +171,226 @@ export function LabelsPage() {
+                      })}
+                    </span>
+                  </td>
+                  <td className="td-actions">
+                    {!label.internal && (
+                      <div className="label-actions-cell">
+                        <button
+                          className="label-menu-button"
+                          onClick={(e) => toggleMenu(label.id, e)}
+                          aria-label="Label actions"
+                        >
+                          <svg width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)